micro-ui/web/micro-ui-internals/packages/modules/common/README.md

```
@@ -53,35 +417,207 @@ const initDigitUI = () => {
+  "order": 1,
+  "tenants": [
+    {
+      "code": "your-tenant-code"
+    }
+  ]
+}
+```
+
+## 🎯 Key Capabilities
+
+### 🔄 Shared Component System
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)